projector-common/src/commonMain/kotlin/org/jetbrains/projector/common/protocol/toClient/WindowData.kt

```diff
@@ -103,4 +103,10 @@ data class WindowData(
   val isAlwaysOnTop: Boolean = false,
   @SerialName("p")
   val parentId: Int? = null,
+  /**
+   * Rendering scale used on the host for this window.
+   * This is intended to help the client size rendering canvas, as `bounds` operates on "user pixels" (as opposed to hardware pixels)
+   */
+  @SerialName("q")
+  val renderingScale: Float = 1f
```

Comment:
```suggestion
  val renderingScale: Float = 1f,
```

By the way, do we need to specify the default value?

Reply:
It's more of a concession to test code, which already relies on other default values. We could drop all default values and update test code accordingly, but that's a separate task.